src/main/java/edu/ucsb/cs56/mapache_search/search/SearchResult.java

```diff
@@ -31,9 +32,19 @@ public String getKind() {
         return this.items;
     }
 
+    public SearchInformation getSearchInformation() {
+        return this.searchInformation;
+    }
+
+    public int getPageCount() {
+        long totalResults = Long.parseLong(searchInformation.getTotalResults());
+
+        return (int) Math.min(Math.ceil(totalResults / 10.0), 10);
```

Comment:
This should have a comment on it referencing the Google API, per Scott's comment